src/main/java/org/cascadebot/cascadebot/music/KaraokeHandler.java

```diff
@@ -0,0 +1,112 @@
+package org.cascadebot.cascadebot.music;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.TextChannel;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.tools.ant.filters.StringInputStream;
+import org.cascadebot.cascadebot.CascadeBot;
+import org.cascadebot.cascadebot.MDCException;
+import org.cascadebot.cascadebot.data.language.Language;
+import org.cascadebot.cascadebot.messaging.MessageType;
+import org.cascadebot.cascadebot.messaging.Messaging;
+import org.jetbrains.annotations.NotNull;
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class KaraokeHandler {
+
+    private static Map<Long, Boolean> karaokeEnabled = new HashMap<>();
```

Comment:
After thinking, I think perhaps it would be better to have a Set<Long> that contains the enabled guilds.